packages/coqide/coqide.8.12.0/opam

```diff
@@ -14,7 +14,7 @@ using the Coq proof assistant.
 
 depends: [
   "coq" {= version}
-  "lablgtk3-sourceview3" {>= "3.1.0"}
+  "lablgtk3-sourceview3" {>= "3.0"}
```

Comment:
I'd have thought this bound was trivially satisfied?

Reply:
See e.g. https://github.com/ocaml/opam-repository/tree/master/packages/lablgtk3-sourceview3

Reply:
Coming from Rust, an unconstrained dependency is a complete no-go... but sure.

Reply:
Thanks. And sorry that we looped back here... (My initial question was still a valid question, but I could have spent more looking up the answer 😅).